### PR TITLE
Examples

### PR DESCRIPTION
**Pull request by toddhgardner**
I archived the previous examples, hopefully we can bring them back as people have questions.

Created 2 new examples for Express and NextJS to show off how the agents all work together.

I also want to rename this repo "Examples" for easier discoverability.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+<p align="center">
+  <a href="https://trackjs.com/" target="_blank" align="center">
+    <img src="https://trackjs.com/assets/images/brand/logo_full_charcoal_red.svg" width="280">
+  </a>
+  <br />
+</p>
+
+# TrackJS Usage and Integration Examples
+
+This repository contains example projects for using and integrating the TrackJS Error Monitoring agents with common JavaScript patterns and frameworks.
```
Comment (J-Griffin) [inline README.md]: Maybe link to express and nextjs examples here like the old one did?

File: nextjs/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.next
+node_modules
```
Comment (J-Griffin) [inline nextjs/.gitignore]: Not that it matters too much, but I didn't see a .gitignore for the  expressjs example. Should there be?

File: README.md (hunk 1 above)
Comment (toddhgardner) [reply README.md]: Sure.

File: nextjs/.gitignore (hunk 2 above)
Comment (toddhgardner) [reply nextjs/.gitignore]: it didnt need one, didnt do anything special.

**Comment by J-Griffin** [comment]
:+1:

**Comment by BrandesEric** [comment]
Good call! I bet we’ll think of plenty of other ones to do too.

**Pull request merged**